neuralmonkey/attention/base_attention.py

```diff
@@ -10,7 +10,7 @@
 ``attention_states`` tensor.  The attention function receives the query tensor,
 the decoder previous state and input, and its inner state, which could bear an
 arbitrary structure of information. The default structure for this is the
-``AttentionLoopState``, which contains a growing array of attention
+``AttentionLoopStateTA``, which contains a growing array of attention
```

Comment:
Z toho názvu není jasné, co to dělá. Navrhuju třeba `AttentionLoopHistory`, abz bylo zřejmé, že to sbírá informace o historii dekódování.